delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeStatisticsCollection.scala

```diff
@@ -0,0 +1,55 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model
+
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.ContextValue
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ProgressStatistics
+import io.circe.syntax._
+import io.circe.{Encoder, JsonObject}
+
+/**
+  * Statistics for a collection of composite view projections
+  *
+  * @param summary     the summary of statistics from the values
+  * @param values the collection of statistics
+  */
+final case class CompositeStatisticsCollection private (summary: ProgressStatistics, values: List[CompositeStatistics])
+
+object CompositeStatisticsCollection {
+
+  private def maxOption[A: Ordering](x: Option[A], y: Option[A]): Option[A] = (x ++ y).maxOption
+
+  /**
+    * Constructor helper from a collection of [[CompositeStatisticsCollection]]
+    */
+  // TODO: It does not make any sense to just take maximums/minimums and the resulting summary it is just misleading.
+  // Summary should be removed from the API response and statistics endpoint should return a SearchResults[CompositeStatistics]
+  def apply(values: List[CompositeStatistics]): CompositeStatisticsCollection = {
+    val (head, tail) =
+      values.headOption.fold(ProgressStatistics.empty -> List.empty[CompositeStatistics])(_.value -> values.tail)
+    val summary      = tail.foldLeft(head) { case (acc, CompositeStatistics(_, _, stats)) =>
```

Comment:
We could get rid of the summary no ? It does not bring useful information as the max does not mean much for most of the fields

Reply:
I agree. It is there just for backwards compatibility. But it is something we can discuss tomorrow morning. If it were for me I would remove it, as I explain in the TODO

Reply:
Fixed